app/routers/dogfinder.py

```diff
@@ -195,6 +207,7 @@ async def add_documents(documentRequest: DocumentRequest):
 
                 # Create a uuid based on the filename
                 data_properties[UUID_FIELD] = generate_uuid5({"filename": document.filename})
+                data_properties[DOG_ID_FIELD] = request_dog_id
```

Comment:
Lets move this to create_data_properties

Reply:
ok